libsolidity/formal/CHC.cpp

```diff
@@ -155,29 +222,105 @@ bool CHC::visit(IfStatement const& _if)
 {
 	solAssert(m_currentFunction, "");
 
+	bool functionCallSeen = false;
```

Comment:
I think
```
bool functionCallSeen = m_unknownFunctionCallSeen;
m_unknownFunctionCallSeen = false;
```
would be easier to read... Maybe even ``unknownFunctionCallWasSeen`` instead of ``functionCallSeen``?